contracts/MembersLibrary.sol

```diff
@@ -65,13 +65,14 @@ library MembersLibrary {
 
     function getMember(address memberStoreAddress, uint index) constant returns (address account, uint state, uint dateAdded) {
         var memberStore = DataStore(memberStoreAddress);
-        if (index < 1) {
+        if (index < 1 || index > memberStore.count()) {
```

Comment:
This condition cannot be here as 'index' is the accountIndex(superset member index) which can be more than the memberIndex(subset member index).